things/thing-7.md

```diff
@@ -0,0 +1,158 @@
+landscape and as yet, is not routinely done by researchers, or expected
+by most journals. What could be done to encourage routine citation of
+research data and software associated with research outputs?
+
+
+Do you have a question?  Want to share a resource?
+- Post to the [Data Librarians Slack group](https://tiny.cc/data-librarians) to connect with the community.
+- Tweet to [@ardc_au](https://twitter.com/ARDC_AU) using hashtag#23things
+
+Go to [Thing 8](thing-8.md)
+Citation metrics for data or to [All Things](index.md)
+
+## Learn more
+
+# Data citation principles
+
+
+The Force11 Joint Declaration of Data Citation Principles are a set of
+principles for citing data. They are based on the premise that data
+citation, like the citation of other evidence and sources, is good
+research practice and is part of the scholarly ecosystem supporting data
+reuse.
+
+Since they were published, the Principles have been endorsed by numerous
+individuals and more than 100 data centres, publishers and societies.
+
+1.  Start by reading the [Force11
+    Principles](https://www.force11.org/group/joint-declaration-data-citation-principles-final)
+2.  Then browse the list of people and organisations that [have endorsed
+    the Principles](https://www.force11.org/datacitation/endorsements)
+
+**Consider:** Given such support and clear direction, why do you think
+data citation has not been uniformly adopted, so far, across all
+disciplines?
+
+Do you have a question?  Want to share a resource?
+- Post to the [Data Librarians Slack group](https://tiny.cc/data-librarians) to connect with the community.
+- Tweet to [@ardc_au](https://twitter.com/ARDC_AU) using hashtag#23things
+
+Go to [Thing 8](thing-8.md)
+Citation metrics for data or to [All Things](index.md)
+
+## Challenge me 
+
+### Data formats and data citation
+
+Are some data formats more likely to be cited?
+
+It has become a lot simpler to mine data and interpret insights in an
+engaging, attractive and easy to understand way. Does this make the data
+more or less accessible, reusable and therefore citable?
+
+1\. Go to [this list, from
+econsultancy.com](https://econsultancy.com/blog/66131-17-visualisation-tools-to-make-your-data-beautiful/),
+of various free and premium tools and platforms for creating interactive
+charts, infographics, maps and word clouds.
+
+2\. Choose one tool and delve into the detail.
+
+3\. If you want to get hands on, work through this [infogram
+tutorial](https://support.infogram.com/hc/en-us/articles/203063586-Video-tutorial-Get-started-with-Infogram- "getting started with infogram")
+to create a data visualisation using your own data or the titanic
+passenger dataset they have provided.
+
+**Consider:**
+
+1\. What are 3 of your likes or dislikes about the tool you explored?
+
+2\. What is your opinion about whether publishing data as a visualisation
+is likely to make the data more or less accessible, reusable and
+therefore citable.
```

Comment:
Add "?" at the end of this sentence. ("?" not contained in original text)

Reply:
Added